app/assets/javascripts/fae/form/_form.js

```diff
@@ -23,8 +23,35 @@ Fae.form = {
       $('.input.file').fileinputer();
     }
 
+    // Mutate DOM to support two column labels for all standard inputs
+    this.make_two_column_labels();
+
     // make all the hint areas
     $('.hint').hinter();
   },
 
+  make_two_column_labels: function() {
+    $('.input label').each(function() {
+      var $element = $(this);
```

Comment:
I'm OK with this, but @jamesmk would like your input. Whenever `var ? = $(this)` is called, `$this` is used.